### ISSUE TITLE
OTM 5.5 release candidates

### ISSUE THREAD

**Issue by bmeesters** [opentripmodel/otm5-change-requests#65]
# Introduction

As done in [OTM5.3](https://github.com/opentripmodel/otm5-change-requests/issues/45) and [OTM5.4](https://github.com/opentripmodel/otm5-change-requests/issues/57) I want to have an overview again of the available _change requests_ that are considered and their status. This makes it easy to view everything that is in scope and make decisions on what's left open.

# Considered Change Requests

Here are all _change requests_ that have been open/created since OTM5.4. They can be divided into _accepted_, _open_ and _rejected_ change requests.

## Accepted change requests

* [42](https://github.com/opentripmodel/otm5-change-requests/issues/42): Improve documentation on constraints.
  **status**: accepted.
  **why**: this is not a change requests on the specification, but a clarification on the documentation on which constraints are relevant when. Therefore easily accepted.

* [51](https://github.com/opentripmodel/otm5-change-requests/issues/51): Make fuelType an enum to avoid parties using different type of values for fuel.
  **status** accepted.
  **why**: Currently having everything open makes it harder to be consistent.

* [52](https://github.com/opentripmodel/otm5-change-requests/issues/52): Add a new `emissionStandardConstraint` to indicate the minimum level required to enter an area or location.
  **status**: accepted.
  **why**: the `emissionType` is present on vehicles, but you cannot communicate that a certain emission standard is required on a certain route, trip, or location. For consistency and completeness it makes sense to add it.

* [59](https://github.com/opentripmodel/otm5-change-requests/issues/59) Add the `transportOrder` as a field on the `consignment` so that the association works both ways.
  **status**: accepted.
  **why**: In OTM5 most enties can point both ways to support multiple use cases. This is currently not the case for knowing which consignments are part of which transport orders. Adding that makes the standard more consistent and complete.

* [60](https://github.com/opentripmodel/otm5-change-requests/issues/60): Add EORI number as a contact detail option.
  **status**: accepted.
  **why**: EORI numbers are a standard within the European Union and used for customs.

* [61](https://github.com/opentripmodel/otm5-change-requests/issues/61): Simplify constraints by merging similar ones.
  **status**: accepted.
  **why**: Constraints are very powerful, but can be complicated to understand. The current proposal adds another constraint (valueBound) that can replace multiple others (speed, weight, size, temperature). This simplifies the usage. The older constraints will be deprecated but remain supported.

* [62](https://github.com/opentripmodel/otm5-change-requests/issues/62) Add a new action `refuel` that can be used for both refueling the vehicle and charging the vehicle. The type of 'fuel' can be used to indicate the difference.
  **status**: accepted.
  **why**: Adding fuel is common operation during the trip. This can be currently only be modeled using a generic action. Adding a new type of action is more clear.

* [64](https://github.com/opentripmodel/otm5-change-requests/issues/64): Add EURALCODE (to ADR?).
  **status**: accepted, needs some refinement

## Open change requests

These change requests are currently open but not yet considered for OTM5.5

* [63](https://github.com/opentripmodel/otm5-change-requests/issues/63): Add transportEquipment to load/unload actions to indicate where something is loaded. Necessary for super lorries.
  **status**: Thomas Kolmans will demonstrate a more generic solution to the same problem.

* [67](https://github.com/opentripmodel/otm5-change-requests/issues/67): Add a new actor role for dealing with 'sub' companies.
  **status**: rejected for now, needs more clarification.

* [66](https://github.com/opentripmodel/otm5-change-requests/issues/66): Fix the documentation so it is accepted in MuleSoft. It seems the openAPI is not entirely correct, with a few changes we can fix that (no changes to the actual specification).
  **status**: check if we can solve it

* [68](https://github.com/opentripmodel/otm5-change-requests/issues/68): After extensive discussion around invoices, this is a smaller change request that only considers adding the product costs to the goods items.
  **status**: likely useful, but no push now.

## Rejected change requests

These change requests were rejected for OTM5.

* [21](https://github.com/opentripmodel/otm5-change-requests/issues/21): Add invoice information to OTM5. We understand the need but ultimately reject the proposal because invoice information is very complicated and not intrinsic part of traffic or logistic operations. [68]((https://github.com/opentripmodel/otm5-change-requests/issues/68) is a smaller and simpler alternative.

* [58](https://github.com/opentripmodel/otm5-change-requests/issues/58): Add `travelDistanceEvent` to indicate how much distance has been covered in a trip or vehicle. This was rejected since it is not a 'pure' event like others. Also there is not really much demand for it.

**Comment by bmeesters** [comment]
I completed the overview & all open change requests are now mentioned here. We can go over this ticket and decide which will make it in, which will be closed, and which can be postponed for now.

**Comment by bmeesters** [comment]
After going over all the tickets we accepted most of them. 

Things to note:
* Thomas Kolmans has a more generic solution for indicating where something is loaded (not only which trailer, but also what placement).
* EURAL code needs some information from Wout before we can add it.
* Marinus Brink is creating a new change request to deal with road works (access constraint).
* The sub company role is currently rejected.

**Comment by bmeesters** [comment]
OTM5.5 has been released so this can be closed.
